README.md

```diff
@@ -2,47 +2,65 @@
+
+Call `npm i && npm start` to run this project in your browser
 
 On a high level, you will be working on several components that form the UI of the animal shelter adoption application. Users can filter for pets by type, and can adopt a pet of their choosing. Once a pet is adopted, they cannot un-adopt it. No backsies!
 
-There are several components that need your attention. All of these components can be found in the `components/` folder. Starting with the smallest one and working our way up:
+There are several components that need your attention. All of these components can be found in the `components/` folder. Starting from the top and working our way down:
 
-### `Pet`
-1. Should have a `pet` prop. Use the attributes in this data to render the pet card correctly. It should show the pet's `name`, `type`, `age` and `weight`. Based on the pet's `gender`, the component also needs to contain either a male (`♂`) or female (`♀`) symbol.
-2. Should have an `isAdopted` prop. Using this prop, render the correct button in the pet's card; if the pet is adopted, show the disabled button. Otherwise, show the primary button to adopt the pet.
-2. Should have an `onAdoptPet` callback prop. This callback prop gets called with the pet's `id` when the user clicks the adopt pet button — _not_ when they click the disabled button!
+### `App`
 
-### `PetBrowser`
-1. Should have a `pets` prop. This is an array of pets that the component uses to render `<Pet />` components.
-1. Should have an `onAdoptPet` prop. This callback prop gets passed to its `<Pet />` children components.
-1. Should have an `adoptedPets` prop. Use this prop to figure out if a pet is adopted or not, and pass that result to the `<Pet />` components in the form of an `isAdopted` prop.
+1.  The app's initial state is already defined. App has two children: the `<Filters />` and `<PetBrowser />` components.
+
+* `<Filters />` needs a **callback** prop, `onFindPetsClick`. When the `<Filters />` component calls `onFindPetsClick`, `<App />` should fetch a list of pets using `fetch()`.
+
+  * The URL of the API is `/api/pets` with an **optional query parameter**.
+  * Use app's `state.filters` to create this parameter
+  * If the `type` is `'all'`, send a request to `/api/pets`
+  * If the `type` is `'cat'`, send a request to `/api/pets?type=cat`. Do the same thing for `dog` and `micropig`.
+  * Finally, set App's `state.pets` so you can pass this data down as props to `<PetBrowser />`
+
+  * **Even though we're using `fetch` here, its responses have been mocked in order to make the tests work properly. That means it's important to use the _exact_ URLs as described above, or your tests will fail!**
+
+* App should pass a **callback** prop, `onChangeType`, to `<Filters />`. This callback needs to update `<App />`'s `state.filters.type`
```

Comment:
maybe this should go before the onFindPetsClick ? Like this happens before you click to find pets. Might help to make things make more sense.